### DISCUSSION TITLE
Question on how to do role mangement for input fields by using directive

### DISCUSSION THREAD

**Discussion by sunilkumarrudragada** [apollographql/apollo-server#6921] [Q&A]
can we add role permissions to input field?
I tried by writing directive on INPUT_OBJECT_FIELD but looks like context won't be available as I need user object from context
Is it possible to handle role permissions to input field by using directive?

**Comment by glasser** [comment]
(I've converted to a discussion because this isn't identifying a bug in Apollo Server but just looking for advice on how to use it.)
